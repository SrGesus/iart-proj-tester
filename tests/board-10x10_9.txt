VD	LH	VE	FD	VD	BB	FD	FC	FD	FE
FB	FC	BB	BC	LV	LV	FE	VC	BD	LV
FB	BE	VE	LV	LV	FE	BD	FE	BC	VE
VC	VD	FE	LH	LV	FC	BE	FE	BD	FC
FE	VD	VD	BD	BB	VE	BB	LV	VB	LV
BE	VB	FD	VB	VB	BB	FD	VB	FD	LV
BB	BB	VD	FB	VB	LH	LV	LH	FD	BB
LV	FB	BE	LH	BE	BE	BD	BD	FE	LV
FD	FC	BC	VB	VD	BC	BC	BD	LV	VB
FB	BC	VB	FD	FD	FD	VE	BB	LV	FD
