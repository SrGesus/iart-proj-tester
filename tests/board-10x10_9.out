VB	LH	VE	FB	VB	BB	FE	FB	FB	FB
FC	FB	BD	BE	LV	LV	FB	VD	BE	LV
FB	BD	VC	LV	LV	FC	BD	FE	BD	VC
VD	VC	FB	LV	LV	FD	BE	FD	BE	FB
FB	VB	VC	BD	BE	VB	BC	LH	VC	LV
BD	VC	FD	VC	VD	BE	FB	VB	FE	LV
BD	BB	VE	FD	VE	LV	LV	LV	FD	BE
LV	FC	BD	LH	BC	BC	BE	BD	FE	LV
FC	FB	BD	VE	VB	BB	BE	BD	LH	VC
FD	BC	VC	FC	FC	FC	VD	BC	LH	FE
